lib/Arch/Sleigh/Arch.h

```diff
@@ -95,13 +95,31 @@ class SingleInstructionSleighContext {
   std::vector<std::string> getUserOpNames();
 };
 
+struct ContextRegMappings {
+
+ private:
+  std::unordered_map<std::string, std::string> context_reg_mapping;
+  std::unordered_map<std::string, size_t> vnode_size_mapping;
+
+ public:
+  ContextRegMappings(
+      std::unordered_map<std::string, std::string> context_reg_mapping,
+      std::unordered_map<std::string, size_t> vnode_size_mapping)
```

Comment:
can you put a comment somewhere that explains what vnode_size_mapping is for? looks like it's set maps to 1 for both ppc and thumb?